pvlib/tests/iotools/test_sodapro.py

```diff
@@ -248,20 +248,21 @@ def test_get_cams_bad_request(requests_mock):
     requests inputs. Also tests if the specified server url gets used"""
 
     # Subset of an xml file returned for errornous requests
-    mock_response_bad = """<?xml version="1.0" encoding="utf-8"?>
+    mock_response_bad_text = """<?xml version="1.0" encoding="utf-8"?>
     <ows:Exception exceptionCode="NoApplicableCode" locator="None">
     <ows:ExceptionText>Failed to execute WPS process [get_mcclear]:
         Please, register yourself at www.soda-pro.com
     </ows:ExceptionText>"""
```

Comment:
I think this error text is out of date.  From the real API, I get this response when supplying an unregistered email:

`<?xml version="1.0" encoding="UTF-8"?>\n<!-- PyWPS 4.5.0 -->\n<ows:ExceptionReport xmlns:ows="http://www.opengis.net/ows/1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.opengis.net/ows/1.1 http://schemas.opengis.net/ows/1.1.0/owsExceptionReport.xsd" version="1.0.0">\n  <ows:Exception exceptionCode="NoApplicableCode" locator="" >\n      <ows:ExceptionText>Process error: Dear User, thank you for your interest in the McClear service provided by MINES ParisTech. The service has been developed with funding from the European Union s Copernicus Programme CAMS project . The data policy imposed by the European Union in CAMS project is that users must be identified. Please,</ows:ExceptionText>\n  </ows:Exception>\n</ows:ExceptionReport>`

Probably we should update to the current API message, right?

Note also that the end of the message seems to be truncated ("Please, " do what?).

Reply:
Oh, I see this error message has already been discussed :)

If it will change again soon, what should we do here?  I don't think it makes sense to test on out of date messages, but it seems that even the current message will soon be out of date.

Reply:
Yes, the message will be changed. 
As far as I understand, because we handle the answer through mock, the real content of the message is not important in the test as long as the format is correctly simulated.